### DISCUSSION TITLE
Is there a CommandTimeout setting in Azure Data Studio

### DISCUSSION THREAD

**Discussion by JustinFrostF5** [microsoft/azuredatastudio#19774] [Q&A] [answered]
I am trying create a database by importing a bacpac file.

The bacpac file is about 142MB in size.

It consistently fails after 10 minutes with the following error:

Import bacpac: An error occurred during deployment plan generation. Deployment cannot continue.
Unable to reconnect to database: Execution Timeout Expired. The timeout period elapsed prior to completion of the operation or the server is not responding.

Is there a setting in ADS to get around this error ?

I have set query timeout and connection timeout to unlimited (0).

Thanks.

**Comment by cheenamalhotra** [comment, marked as answer]
Yes, there is.
You can find it in Advanced connection dialog:

<img src="https://user-images.githubusercontent.com/13396919/226822884-ab10cecb-2ff5-4bf0-9d09-9930309af793.png" width=500 />

It defaults to 30 seconds.
